sdk/storage/azure-storage-file-datalake/tests/test_directory.py

```diff
@@ -1095,6 +1098,29 @@ def test_get_properties(self, datalake_storage_account_name, datalake_storage_ac
         self.assertIsNotNone(properties.metadata)
         self.assertEqual(properties.metadata['hello'], metadata['hello'])
 
+    @DataLakePreparer()
+    def test_directory_encryption_scope(self, datalake_storage_account_name, datalake_storage_account_key):
```

Comment:
Can we adjust the naming here to something like `test_directory_encryption_scope_from_file_system`. We are testing that the directory gets the default encryption scope set on the file system. Later we will be adding setting an ecnryption scope directly on a directory which we would probably want to use this name for that.

Reply:
I completely forgot about that point. Will change the name!